README.md

```diff
@@ -178,13 +177,6 @@ You can run it using
 $ aegir coverage
 ```
 
-To auto publish coverage reports from Travis to Coveralls add this to
```

Comment:
This should still be here, but with Codecov rather than Coveralls.

Reply:
Done with a subsequent commit.